js_src/rdm.js

```diff
@@ -340,20 +340,26 @@ angular.module('rdmApp', [])
      var token = tokens[j];
      var hex_suffix_match = token.match(/^([\da-fA-F]{1,2})h$/);
      if (hex_suffix_match) {
-      binary_data.push(parseInt(hex_suffix_match[0], 16));
+      binary_data.push(parseInt(hex_suffix_match[1], 16));
       continue;
      }
 
      var hex_prefix_match = token.match(/^0x([\da-fA-F]{1,2})$/);
      if (hex_prefix_match) {
-      binary_data.push(parseInt(hex_prefix_match[0], 16));
+      binary_data.push(parseInt(hex_prefix_match[1], 16));
+      continue;
+     }
+
+     var char_match = token.match(/^'(.)'$/);
+     if (char_match) {
+      binary_data.push(char_match[1].charCodeAt());
```

Comment:
Apparently 0 is the default.

I think I'm going to re-write the parser anyway.